exception_al/workshops/views.py

```diff
@@ -31,6 +32,9 @@ def post(self, request):
             return Response(serializer.data, status=status.HTTP_201_CREATED)
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
     
+
+    
+    
```

Comment:
For below WorkshopDetailView, modify permissions to IsOwnerOrReadonly permission that you created. You'll need to import the custom permission from permissions.py

Reply:
Done!!
Imported custom permissions from permissions.py
modified permissions to IsAuthenticatedOr ReadOnly, CustomPermission